.github/workflows/release.yml

```
@@ -21,31 +43,28 @@ jobs:
-        uses: softprops/action-gh-release@v2
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          name: Release ${{ github.ref_name }}
-          tag_name: ${{ github.ref }}
-          body_path: CHANGELOG.md
-          draft: false
-          prerelease: false
-          files: |
-            dist/**
-            CHANGELOG.md
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)